db/models/points.js

```diff
@@ -0,0 +1,149 @@
+const BaseService = require('../common/service.js');
+const knex = require('../knex.js');
+const knexPostgis = require("knex-postgis");
+const wkx = require('wkx');
+const Buffer = require('buffer').Buffer;
+const geoHash = require('../../app/lib/geoHash');
+
+const st = knexPostgis(knex);
+
+class Service extends BaseService {
+
+  /**
+   * Fetch All Points and run through Redaction.
+   *
+   * @method _getRedactedPoints
+   * @param {String} case_id
+   * @return {Array}
+   */
+  async fetchRedactedPoints(case_id) {
+    if (!case_id) throw new Error('Case ID is invalid')
+
+    const points = await this.find({ case_id })
+    if (points) {
+      return this._getRedactedPoints(points)
+    }
+    throw new Error('Could not find redacted points.')
+  }
+
+  async createRedactdPoint(caseId, point) {
+    let record = {};
+    let hash = await geoHash.encrypt(point)
```

Comment:
This (and the same function call in the update action below) takes care of [this ticket](https://pathcheck.atlassian.net/browse/PLACES-241), right?

Reply:
Correct.